### ISSUE TITLE
Pimp the visually impaired interface

### ISSUE THREAD

**Issue by minj** [minj/foxtrick#1117]
Original [issue 1117](https://code.google.com/p/foxtrick/issues/detail?id=1117) created by [minj](mailto:4mr.minj@gmail.com) on 2013-06-17T23:53:24.000Z:

Needs analogous adequate features. Should be copy/paste at least in part. But would suck to maintain. I guess the corresponding modules need to be interface-agnostic
